src/App.js

```diff
@@ -1,20 +1,24 @@
-import { Route, Routes } from 'react-router-dom';
-import Navigation from './components/navBar';
-import Home from './pages/books';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Navigation from './components/Navbar/navBar';
+import BookPage from './pages/bookPage';
 import Categories from './pages/categories';
+import Books from './pages/books';
 import './App.css';
 
 function App() {
   return (
-    <>
-      <Navigation />
-      <div>
-        <Routes>
-          <Route path="/" element={<Home />} />
-          <Route path="/categories" element={<Categories />} />
-        </Routes>
-      </div>
-    </>
+    <div className="wrapper">
+      <Router>
+        <Navigation />
+        <div>
+          <Routes>
+            <Route path="/" element={<Books />} />
+            <Route path="/bookPage" element={<BookPage />} />
+            <Route path="/categories" element={<Categories />} />
+          </Routes>
+        </div>
+      </Router>
+    </div>
```

Comment:
- [ ] Please consider following as closely as possible the given design:

| Template |
|---|
|![image](https://github.com/pushkar1995/react-bookstore-app/assets/77069011/26d86faa-61cf-4d4a-b41a-b3cd4f3cd672)|

|Your design|
|---|
|![image](https://github.com/pushkar1995/react-bookstore-app/assets/77069011/2af7ce6b-1cad-4048-9e7d-0ef5212d2d31)|


- [ ] 🚧 Navbar should be at the top, and with a border bottom.  🔧 
- [ ] 🚧 Please consider using the same font sizes and widths. 🔧 
- [ ] 🚧 The content section (where the books are) should have a light gray background 🔧 
- [ ] 🚧 Each book should have its container with a white background and border. 🔧 
- [ ] 🚧 There are dividers for each action button (comments, remove, edit) and other divider between the circular progress and the chapters 🔧 
- [ ] 🚧 All books should be centered, and aligned with the navbar and form. 🔧 
- [ ] 🚧 The inputs and the button should be thinner, to match the design 🔧 

Please consider applying those changes! 🤓 🔧